db/migrate/20250318164438_add_preapp_disclaimer_to_local_authority.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddPreappDisclaimerToLocalAuthority < ActiveRecord::Migration[7.2]
+  def change
+    add_column :local_authorities, :preapp_disclaimer, :string
```

Comment:
Not a big fan of this field name - wondering whether we could take advantage of @EGiataganas work on LA application type overrides and add `disclaimer` as an optional field to the application types table?